pubsub/google/cloud/pubsub_v1/subscriber/client.py

```diff
@@ -124,6 +124,9 @@ def subscribe(self, subscription, callback=None, flow_control=()):
         Returns:
             ~.pubsub_v1.subscriber.consumer.base.BaseConsumer: An instance
                 of the defined ``consumer_class`` on the client.
+
+        Raises:
+            :exc:`TypeError` if callback is not callable.
```

Comment:
I believe the "correct" way to do this with a Google-style docstring is:

```
Raises:
    TypeError: If ``callback`` is not callable.
```

(I added the backticks around `callback` so that it would be in code font, that's just a personal preference.)